segmentstore/server/src/main/java/io/pravega/segmentstore/server/containers/StreamSegmentContainer.java

```diff
@@ -99,13 +101,18 @@
 import lombok.extern.slf4j.Slf4j;
 import lombok.val;
 
+import static io.pravega.segmentstore.contracts.AttributeUpdateType.Replace;
```

Comment:
These classes are already imported; let's try to keep the code clean by not statically importing members of those classes.

Reply:
Fixed.